mifeille/EpicMail
# API Endpoints [Finishes #164770448]

What does this PR do?

This PR sets up API Endpoints .

Description of Task to be completed?

Create a user account, Sign in a user, Get all received emails for a user, Get all unread emails for a user,Get all emails sent by a user , Get a specific user’s email, Send email to individuals 
 and Delete an email in a user’s inbox API Endpoints have been created.

How should this be manually tested?

Clone the branch
Install all dependencies using npm install
Run the server by running this command (npm start) then go to your postman log in with a token choose POST and type api/v2/messages to write a message, choose GET and type api/v2/messages to see a user  received messages, choose GET and type api/v2/messages/unread to see a user unread messages, choose GET and type api/v2/messages/sent to see a user sent messages, choose DELETE and type api/v2/messages/id(the id of the message you want to delete) to delete a message, choose GET and type api/v2/messages/id (the id of the message you are searching for) to search for a message.

Any background context you want to provide?
N/A

What are the relevant pivotal tracker stories?
Challenge 2 API Endpoints. 164770448

Screenshots (if appropriate)
N/A

Questions:
N/A
